README.md

```diff
@@ -60,9 +74,9 @@ test('Some test case', function(assert) {
     }
   };
 
-  visit('/');
-  a11yAudit(axeOptions);
-  andThen(() => assert.ok(true, 'no a11y errors found!'));
+  await visit('/');
+  await a11yAudit(axeOptions);
+  assert.ok(true, 'no a11y errors found!'));
```

Comment:
There's one too many parens here: `assert.ok(true, 'no a11y errors found!'));`

Congrats on the new baby!!!

Reply:
Oops, I’ll get that fixed this week or weekend